Snowflake.Data/Client/SnowflakeDbDataReader.cs

```diff
@@ -81,6 +81,49 @@ public override bool IsClosed
 
         public override int RecordsAffected => resultSet.CalculateUpdateCount();
 
+        public override DataTable GetSchemaTable()
+        {
+            if (this.FieldCount == 0)
+            {
+                // No result
+                return null;
+            }

```

Comment:
We should probably return an empty dataTable instead of null. Although I don't think there is a possibility that no column is returned.

Reply:
That's fine with me.